tests/e2e/framework/kubernetes.go

```diff
@@ -280,10 +286,18 @@ func newKubeInfo(tmpDir, runID, baseVersion string) (*KubeInfo, error) {
 		if baseVersion != "" {
 			remoteI.localPath = filepath.Join(releaseDir, "/bin/istioctl")
 		}
-		aRemote = NewAppManager(tmpDir, *namespace, remoteI, remoteKubeConfig)
+		if *splitHorizon {
+			// For remote Istio configuration in a split horizon test we don't wait
+			// for the deployments because they won't be able to connect to Pilot
+			// until their service is added by watching the remote k8s api. Adding
+			// the remote k8s api watcher is done later on in the test.
+			aRemote = NewAppManager(tmpDir, *namespace, remoteI, remoteKubeConfig, false)
```

Comment:
I understand while you initially can't check the remote deployment,  but it might be good to check the remote deployment add some point.   If it doesn't come up the test will fail either way so it isn't critical but it would both provide better failure insight and also likely fail sooner.